src/vs/workbench/services/remote/browser/remoteAgentService.ts

```diff
@@ -39,11 +40,18 @@ class RemoteConnectionFailureNotificationContribution implements IWorkbenchContr
 		@IRemoteAgentService remoteAgentService: IRemoteAgentService,
 		@IDialogService private readonly _dialogService: IDialogService,
 		@IHostService private readonly _hostService: IHostService,
+		@ITelemetryService telemetryService: ITelemetryService,
 	) {
+		type ConnectFailedClassification = {
+			owner: '';
+			comment: '';
+		};
+		type ConnectFailedEvent = {};
 		// Let's cover the case where connecting to fetch the remote extension info fails
 		remoteAgentService.getRawEnvironment()
 			.then(undefined, (err) => {
 				if (!RemoteAuthorityResolverError.isHandled(err)) {
+					telemetryService.publicLog2<ConnectFailedEvent, ConnectFailedClassification>('gitpod.connectFailed', {});
```

Comment:
Not neede, this is covered in `src/vs/workbench/contrib/remote/common/remote.contribution.ts`